src/main/java/uk/gov/hmcts/reform/sendletter/config/AuthConfiguration.java

```diff
@@ -21,7 +21,7 @@ public AuthTokenValidator tokenValidator(ServiceAuthorisationApi s2sApi) {
 
     @Bean
     @ConditionalOnProperty(name = "idam.s2s-auth.enabled", havingValue = "false")
-    public AuthTokenValidator tokenValidator() {
+    public AuthTokenValidator tokenValidatorStub() {
```

Comment:
although signatures were different, method names were the same for both beans, which was confusing Spring